common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -199,6 +199,13 @@
     return get_value('SHOW_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False))
 %></%def>
 
+<%def name="show_footer_language_selector(api_setting)"><%
```

Comment:
I'd rather not put an edX-specific piece of logic into this very general class. Is there a footer.py that it could live in?